src/components/views/right_panel/GroupHeaderButtons.js

```diff
@@ -50,6 +52,14 @@ export default class GroupHeaderButtons extends HeaderButtons {
         }
     }
 
+    _onMembersClicked() {
```

Comment:
I believe event handlers need to use ` _onMembersClicked = () => {` style property initializers to have the right `this` value, see [code style](https://github.com/matrix-org/matrix-react-sdk/blob/develop/code_style.md#react).

Reply:
Oh, oops, you bound them! 😅 So, should be fine.